pkg/install/v1alpha1/jiva_volume_0.7.0.go

```diff
@@ -462,6 +458,7 @@ spec:
         vsm.openebs.io/targetportals: {{ .TaskResult.readlistsvc.clusterIP }}:3260
         openebs.io/controller-ips: {{ .TaskResult.readlistctrl.podIP | default "" | splitList " " | first }}
         openebs.io/cluster-ips: {{ .TaskResult.readlistsvc.clusterIP }}
+        openebs.io/controller-node-name: {{ .TaskResult.readlistctrl.nodeName | default "" }}
```

Comment:
call it target-node-name

Reply:
done used ```targetNodeName``` instead as facing issue with target-node-name. I think template doesn't support it.